src/fxcorrelator_bengops.py

```diff
@@ -1355,13 +1363,13 @@ def spead_issue_all(corr, beams=[], from_fpga=True):
     :param from_fpga:
     :return:
     """
-    spead_static_meta_issue(corr, beams=beams)
+    spead_static_meta_issue(corr, beams=beams)  # fxcorrelator.spead_issue_meta
     spead_passband_meta_issue(corr, beams=beams)
-    spead_destination_meta_issue(corr, beams=beams)
-    spead_time_meta_issue(corr, beams=beams)
-    spead_eq_meta_issue(corr, beams=beams)
-    spead_cal_meta_issue(corr, beams=beams, from_fpga=from_fpga)
-    spead_labelling_issue(corr, beams=beams)
+    spead_destination_meta_issue(corr, beams=beams)  # fxcorrelator.spead_issue_meta
```

Comment:
This one sets rx_udp_port and rx_udp_ip_str. In fxcorrelator.spead_issue_meta these fields output_destination_port and output_destination_ip are commented out.